Jenkinsfile

```diff
@@ -36,6 +36,7 @@ spec:
           strategy: ephemeral
   transports:
     qdr:
+      auth: none
```

Comment:
See https://github.com/infrawatch/service-telemetry-operator/pull/492 for an explanation of why we can't currently test with auth enabled